src/service/video-manager-impl.js

```diff
@@ -376,10 +376,14 @@ class VideoEntry {
    * @private
    */
   videoPaused_() {
-    const trackingVideo = assertTrackingVideo(this.video);
-    if (trackingVideo &&
-        trackingVideo.getCurrentTime() !== trackingVideo.getDuration()) {
-      this.analyticsEvent_(VideoAnalyticsType.PAUSE);
+    const v = this.video;
+    if (isTrackingVideo(v)) {
+      const trackingVideo =
```

Comment:
I was going to remove the extra interface in a future PR but I suppose I can knock it out it in this one